### PR TITLE
Changed URI for stellate and added csv data from pig heart

### PR DESCRIPTION
**Pull request by Tehsurfer**
I noticed that I missed this URI, so unfortunately the search and pre-packaged results are currently linking to different states of the app. 

This update modifies the pre-packaged-results to also use the stellate directory navigation, now matching mapcore-augmented-results

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -29,6 +29,13 @@ exports.mapcore_pre_packaged_results_module = function()  {
         "Description": "Pig heart electromechanical data from the Shivkumar/Ardell group is displayed on a 3D pig heart scaffold. The deforming geometry of the pig heart is fitted through the cardiac cycle with the pig heart scaffold. Electrophysiological measurements from two 64-electrode patches, one on the left ventricle (LV) and one on the right ventricle (RV), are fitted as time-varying fields via scaffold parameters and shown on the LV and RV epicardial surfaces, respectively, of the beating heart model.",
         "Example Image": "",
         "Scaffold": {'uri': 'https://mapcore-bucket1.s3-us-west-2.amazonaws.com/ISAN/scaffold/use_case1/fitted_heart_metadata.json', 'species': 'Porcine', 'organ': 'heart', 'annotation': 'UBERON:0000948'}
+      },
+       {
+        "BlackfynnID": "",
+        "Dataset Title": "P1760 IVC Occ trimed.csv",
```
Comment (hsorby) [inline src/index.js]: I think this title should be changed, I don't thing the .csv is needed for instance.

File: src/index.js [hunk 2]
```diff
@@ -29,6 +29,13 @@ exports.mapcore_pre_packaged_results_module = function()  {
...
+        "Description": "Electrophysiological data from the Shivkumar/Ardell group. This data is from an electrode array placed on the surface of the heart while the heart undergoes venticular occlusion",
```
Comment (hsorby) [inline src/index.js]: Check spelling, probably worth mentioning that its a porcine heart.

Comment (Tehsurfer) [reply src/index.js]: Thanks Hugh, just fixed the spelling and reworded to mention it is porcine

File: src/index.js (hunk 1 above)
Comment (Tehsurfer) [reply src/index.js]: I've now switched to 

> P1760 - Intraventricular Occlusion of Pig Heart

**Pull request merged**